### ISSUE TITLE
Modals with DropdownMenuItems doesn't close in one click

### ISSUE THREAD

**Issue by harshit078** [twentyhq/twenty#9102]
## Bug Description

- Modals like Address, ARR, Currency or any other custom field requires 2 clicks to close the modal.
- Page overflow in mobile viewports

## Current Behaviour

https://github.com/user-attachments/assets/5161ccdb-affc-4893-b9e6-dbb8019ee416

https://github.com/user-attachments/assets/e55ccde1-de57-4543-9bf0-0ae4212c670b

## Expected behavior

- Modals with dropdownInput should close in oneclick
- Overflow shouldn't occur for mobile viewports

**Comment by MrSuddenJoy** [comment]
Viewport HAVE to be detected upon scriptload or risk losing compatibility ([source](https://www.w3.org/)).

**Comment by charlesBochet** [comment]
@Bonapara @lucasbordeau this is not a bug to me, from an eng perspective at least this is an healthy trade off.
happy to discuss it further

**Comment by Bonapara** [comment]
I agree with @charlesBochet; the double-click is the desired behavior.
